### PR TITLE
Created Test_#8

### PR DESCRIPTION
**Pull request by lvrub**
@extsoft please review my test

[Test](https://github.com/lvrub/Comments/blob/issue_%2311/src/main/java/net/commments/Test8.java)

### CODE REVIEW

File: src/main/java/net/commments/Test8.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    }
+
+    @BeforeMethod
+    public void createDriver() {
+//     driver[0] = new FirefoxDriver();
+        driver[0] = new ChromeDriver();
+    }
+
+//    @AfterMethod
```
Comment (extsoft) [inline src/main/java/net/commments/Test8.java]: Uncomment this code.

File: src/main/java/net/commments/Test8.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+//import org.openqa.selenium.firefox.FirefoxDriver;
```
Comment (extsoft) [inline src/main/java/net/commments/Test8.java]: Please remove commented code.

File: src/main/java/net/commments/Test8.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    @Test
+    public void test() {
+        this.driver().get("http://commentssprintone.azurewebsites.net");
+        this.driver().findElements(By.name("SelectedId")).get(0).click();
+        Select SelectAnAction = new Select(this.driver().findElement(By.id("commandSelect")));
+        SelectAnAction.selectByIndex(2);
+
+        String table = this.driver().findElement(By.xpath("//*[@id=\"main\"]/div/div[5]/form/table/tbody/tr[1]/td[4]")).getText();
+        assertEquals(table, "V");
+//        System.out.println(table);
```
Comment (extsoft) [inline src/main/java/net/commments/Test8.java]: Please remove commented code.

File: src/main/java/net/commments/Test8.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        Statuse1.selectByIndex(1);
+        this.driver().findElements(By.id("applybutton")).get(0).click();
+        final String page1 = this.driver().getPageSource();
+        MatcherAssert.assertThat("Comment Text 0 is present", !page1.contains("Comment Text 0"));
+
+    }
+
+    @BeforeMethod
+    public void createDriver() {
+//     driver[0] = new FirefoxDriver();
```
Comment (extsoft) [inline src/main/java/net/commments/Test8.java]: Please remove commented code.

Comment (lvrub) [reply src/main/java/net/commments/Test8.java]: @extsoft  Done

**Pull request merged**